bitnami/wordpress/templates/_helpers.tpl

```diff
@@ -22,11 +22,34 @@ Return the proper image name (for the metrics image)
 {{- include "common.images.image" (dict "imageRoot" .Values.metrics.image "global" .Values.global) -}}
 {{- end -}}
 
+{{/*
```

Comment:
If it uses the common library, then you can use something similar to what we do in Harbor

```
{{/*
Return the proper image name (for the init container volume-permissions image)
*/}}
{{- define "harbor.volumePermissions.image" -}}
{{- include "common.images.image" ( dict "imageRoot" .Values.volumePermissions.image "global" .Values.global ) -}}
{{- end -}}
```

Reply:
Hi @javsalgar ,

Thanks for the feedback! I applied your suggested changes and updated the PR.